librz/include/rz_il/rz_il_opbuilder_begin.h

```diff
@@ -86,6 +86,10 @@
 #define FPOW(rmode, flx, fly)      rz_il_op_new_fpow(rmode, flx, fly)
 #define FMAD(rmode, flx, fly, flz) rz_il_op_new_fmad(rmode, flx, fly, flz)
 
+// TODO: add `feq` as prime operator in fbasic
+// https://smtlib.cs.uiowa.edu/theories-FloatingPoint.shtml
+#define FEQ(flx, fly) AND(FORDER(flx, fly), FORDER(fly, flx))
```

Comment:
@brightprogrammer I remember you asked about that

Reply:
Yes I did, there was no equality check for floating pt numbers.